lib/puppet/provider/reboot_notify/notify.rb

```diff
@@ -11,7 +11,7 @@ def exists?
     @records = {}
 
     begin
-      @records = PSON.parse(File.read(@target))
+      @records = JSON.parse(File.read(@target))
     rescue
```

Comment:
Would rather see rescue of exceptions raised by JSON.parse() and File.read(), instead of blanket exception.  For example,

raise JSON::JSONError,ArgumentError,TypeError,SystemCallError